### ISSUE TITLE
Mattekar always evades high Missile attack

### ISSUE THREAD

**Issue by KeithAC** [ACEmulator/ACE#1236]
### Bug Report or Feature Request (mark with an `x`)
```
- [x ] bug report -> please search issues before submitting
- [ ] feature request
```

### Repro steps.
Equip Missile Weapon - Attack Mattekar 

### The log given by the failure.
pcap attached

### Desired functionality.
Mattekar should be able to be hit with Missile weapons using High

Appears at this time they evade 100% of the time while using high attack 
Tested with: Dire, Great, Dread

https://drive.google.com/open?id=1icDd3TXPtfglZkSJlu4Wa_B9NCgPNyy4

**Comment by gmriggs** [comment]
So this one is a real head scratcher...

In the data as of PY16, all Mattekars have no 'high' body parts, ie. their head is at medium height. There are similar monsters with this setup, ie. Phyntos Wasps and Virindi both have no 'low' body parts, so low attacks will always be evaded there.

I remember not being able to hit Phyntos Wasps low, but I don't remember not being able to hit Mattekars high...

I double checked in GDLE, and in there, the logic starts out the same... no high body parts are found for the Mattekar, but then it has a special exception case that runs afterwards, if no body parts are found at that height... it then selects a random quadrant for that height, and tries to perform the attack there.

This has the effect of being able to hit Mattekars high, but it also breaks other things like Phyntos Wasps and Virindi, who can no also be hit with low attacks

So I'm unsure what the proper fix for this should be... it definitely feels odd not being able to hit Mattekars high (esp. when its something like a giant 10 ft. Hoary Mattekar), but according to the data, they have no high body parts... not sure if there is some extra step that could be added to account for this, that would fix Mattekars without breaking Phyntos Wasps / Virindi...

**Comment by gmriggs** [comment]
Confirmed in retail PCAPs that Mattekars could be attacked high

**Comment by gmriggs** [comment]
Fixed in World DB 0.9.12:
https://github.com/ACEmulator/ACE-World-16PY-Patches/releases/download/v0.9.12/ACE-World-Database-v0.9.12.sql.zip
